### PR TITLE
Rename fname to socket_path in SessionManager and ipc.py

### PR DESCRIPTION
**Pull request by DorianGray**

### CODE REVIEW

File: libqtile/ipc.py [hunk 1]
```diff
@@ -107,18 +107,18 @@ def pack(msg: Any, *, is_json: bool = False) -> bytes:
...
         ----------
-        fname : str
+        socket : str
             The file path to the file that is used to open the connection to
             the running IPC server.
         is_json : bool
             Pack and unpack messages as json
         """
-        self.fname = fname
+        self.socket = socket
```
Comment (tych0) [inline libqtile/ipc.py:121]: How about socket_path instead, since this isn't the actual socket?

Comment (DorianGray) [reply libqtile/ipc.py:121]: fair, will just require some reformatting

**Comment by DorianGray** [comment]
done

**Comment by tych0** [comment]
Thanks!

**Pull request merged**